source/playbooks/refining-and-improving.rst

```diff
@@ -23,7 +23,7 @@ Retrospectives help teams identify areas of improvement in workflows. Each time
 Configuring retrospectives before a run
 ----------------------------------------
 
-Open Playbooks and select the **Playbooks** tab. Select the playbook and then select **Edit**. Select **Templates** and scroll down to the **Retrospective Reminder Interval** and **Retrospective Template** fields.
+Open Playbooks and select the **Playbooks** tab. Locate the playbook you want to modify, select the **...** icon under **Actions** and then select **Edit**. In the next screen, select **Retrospective**. Move the toggle to **Enable Retrospective**.
```

Comment:
@cwarnermm, I've added a flow with one less click to it. I don't have a proposal in mind around how to improve the flow but I know refactoring work is in progress so I'll keep an eye on that and see where we can improve this and other flows.

Reply:
This is a great update, @justinegeffen!